Content.Server/GameTicking/Rules/NukeopsRuleSystem.cs

```diff
@@ -495,7 +495,7 @@ private void DoRoundEndBehavior(NukeopsRuleComponent component)
                 }
                 else
                 {
-                    _roundEndSystem.RequestRoundEnd(TimeSpan.FromMinutes(5), null, false, component.RoundEndTextShuttleCall);
+                    _roundEndSystem.RequestRoundEnd(TimeSpan.FromMinutes(5), null, false, component.RoundEndTextShuttleCall, "Central Command");
```

Comment:
is that supports localisation?

Reply:
Yes. The only thing that I probably shouldn't pass that as ID.